package.json

```
@@ -0,0 +1,33 @@
+{
+    "name": "@geode/opengeodeweb-back",
+    "scripts": {
+        "json": "node generate_schemas.js"
+    },
+    "dependencies": {
+        "glob": "^10.3.10"
+    },
+    "exports": "./schemas.json",
+    "version": "0.0.3",
```

```suggestion
    "version": "0.0.0-semantically-released",
```
better to tell that this is not the real version number